node_modules/col-whiteboards/data/whiteboardToPng.js

```diff
@@ -42,15 +42,17 @@ var init = function() {
 
     // Create a PNG stream for these elements
     generatePng(whiteboardElements, function(err, data, dimensions) {
+      // According to https://nodejs.org/api/process.html, "rather than calling process.exit() directly,
+      // the code should set the process.exitCode and allow the process to exit naturally"
       if (err) {
         console.error(err);
```

Comment:
One thing that might be useful to do here is to hook this up to the standard SuiteC logging to ensure that this log message can be found in the application log.